demo/chbench/mzcompose.yml

```diff
@@ -318,6 +323,78 @@ mzconduct:
           -c /etc/peeker/materialized-sources.toml
           -q loadtest
 
+    # Run a test that populates Kafka with data from chbench for a set amount of time
+    # After the first timer expires, stop chbench
+    # Record offsets ingested by Materialize and the answers to our count(*) views
+    # Restart Materialized and measure time until it catches up to previous views
+    ingest-performance-benchmark:
+      steps:
+      - step: workflow
+        workflow: bring-up-source-data-postgres
+      # start the dashboard before the load data get started so that we have data from
+      # the beginning of time
+      - step: start-services
+        services: [dashboard]
+      - step: workflow
+        workflow: heavy-load-postgres
+      # Kafka connect / debezium takes forever to initialize (well, 5 minutes or so)
+      - step: sleep
+        duration: 300
+      - step: run
+        service: peeker
+        command: --only-initialize --queries ${PEEKER_QUERIES:-count-sources}
+      - step: ensure-stays-up
+        container: dashboard
+        seconds: 1
+      # Populate Kafka topics with sufficient data that Materialize will take a decent amount of
+      # time to catch up
+      - step: sleep
+        duration: 64800
```

Comment:
Is 18 hours the intended amount of time?

Reply:
Yeah, it was intended as a "run this for a good amount of time because materialized is much faster reading the data than we are at generating the data".